### PR TITLE
docs: update installation video with cloud

### PR DESCRIPTION
**Pull request by alelthomas**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

File: docs/docs/articles/getting-started-overview.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ In this section you will:
 2. [Install the Testkube Agent](./step2-installing-cluster-components.md).
 3. [Creating your first Test](./step3-creating-first-test.md).
 
-You can also see the full installation video from our product experts: [Testkube Installation Video](https://www.youtube.com/watch?v=bjQboi3Etys):
+You can also see the full installation video from our product experts:
 
-<iframe width="100%" height="315" src="https://www.youtube.com/embed/ynzEkOUhxKk" title="YouTube Tutorial: Getting started with Testing in Kubernetes Using Testkube" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/YnR5yOO8C4s?si=3xdK77wcQVpJQrwt" title="YouTube Tutorial: Installing Testkube Cloud" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
 In summary, you now have Testkube setup in your Kubernetes cluster, ready to discover the Testkube features:
```
Comment (jmorante-ks) [inline docs/docs/articles/getting-started-overview.md:13]: We can also remove all the rest of the content of this page below your video, it is not required there.

It will be included at the OSS installation page once it is defined by Bruno, because the text below the video doesn't make reference to TestKube Cloud, it assumes an OSS installation and that's why at the line 18 even recommends to switch to Cloud :)

**Review by jmorante-ks — CHANGES_REQUESTED** [review]
Thanks

File: docs/docs/articles/getting-started-overview.md (hunk 1 above)
Comment (jmorante-ks) [reply docs/docs/articles/getting-started-overview.md:13]: In other words, remove lines from 13 till the end of the file.

**Review by jmorante-ks — APPROVED** [review]
Thanks @alelthomas

File: docs/docs/articles/getting-started-overview.md (hunk 1 above)
Comment (alelthomas) [reply docs/docs/articles/getting-started-overview.md]: updated and removed lines
